targets/targets.json

```diff
@@ -2366,6 +2386,11 @@
         "bootloader_supported": true,
         "overrides": {
             "network-default-interface-type": "ETHERNET"
+        },
+        "supported_c_libs": {
+            "arm": ["std", "small"],
+            "gcc_arm": ["std", "small"],
```

Comment:
We know there are issues with this board. So we should not enable small libraries.